projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -9,7 +9,7 @@
         class="cx-product-image"
         [container]="product.images?.PRIMARY"
         format="product"
-        [alt]="product.summary"
+        [alt]="getMediaAltText()"
```

Comment:
Not completely sure if this would work, but to avoid adding logic to the component, maybe try:

[alt]="{{ product.summary ?? product.name }}"

Reply:
I will also go this way -> [alt]="{{ product.summary ?? product.name }}"